Sources/TracingInstrumentation/Span.swift

```diff
@@ -353,7 +363,33 @@ public struct SpanAttributes: Equatable {
         where Namespace: SpanAttributeNamespace {
         SpanAttribute.__namespace[keyPath: dynamicMember]
     }
+}
+#else
```

Comment:
why the entire type? can we not #if only the "offending" subscript?

Reply:
It's slightly more than just the subscript. Prior to 5.2 we e.g. won't need `@dynamicMemberLookup` on `SpanAttributes`, right? Also, `subscript(_ name: String)` in the case of < 5.2 doesn't need to handle `__namespace`.